assignments/prototypes.js

```diff
@@ -15,14 +15,30 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
+function gameObject(info){
```

Comment:
Remember that convention is that the first letter of the names of constructors/classes are capitalized, like `GameObject`, `CharacterStats`, and `Humanoid`, which in your case is already capitalized.